wicket-core/src/main/java/org/apache/wicket/settings/RequestCycleSettings.java

```diff
@@ -244,6 +244,16 @@ public List<IResponseFilter> getResponseFilters()
 		}
 	}
 
+	/**
+	 * Allows to clear response filters.
+	 */
+	public void clearResponseFilters()
```

Comment:
Instead of adding new public API methods I'd suggest to use custom RequestCycleSettings class that does nothing at org.apache.wicket.settings.RequestCycleSettings#addResponseFilter()

Reply:
Done but the factory method needs to be non final?

Reply:
https://github.com/apache/wicket/pull/655/commits/f5edc91f45e7f9f4968b82604b2f7c4b14b77038